README.md

```diff
@@ -53,7 +101,7 @@ rx --token-environment list applications
 
 ```
   build-deploy Will trigger build-deploy of a Radix application
-  follow       Follow Radix resources
+  get          Get Radix resource
```

Comment:
Maybe we should remove this block entirely, as the users can for them selves see the supported commands and if we forget the documentation might be wrong